src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -318,6 +324,79 @@ def fuzz(self,
     return LibFuzzerCommon.fuzz(self, corpus_directories, fuzz_timeout,
                                 artifact_prefix, additional_args)
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner,LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+  def __init__(self, executable_path, default_args=None):
+    # This code assumes the following layout for files in the storage bucket:
+    # * multiboot.bin
+    # * fvm.blk
+    # * fuchsia-ssh.zbi
+    # * qemu/*
+    # * .ssh/*
+    local_path = os.getcwd() + "/"
+    resources_path = local_path + "fuchsia_on_clusterfuzz_resources_v1"
+
+    try:
+      os.mkdir(resources_path)
+    except OSError as e:
+      if e.errno == errno.EEXIST:
+        pass
+    subbed_fuchsia_gsutil_command = [param.replace("{fuchsia_resources_path}", environment.get_value('FUCHSIA_RESOURCES_PATH', ''))
```

Comment:
Please use `copy_file_to` from `src/python/google_cloud_utils/storage.py` instead of calling gsutil directly.

Reply:
I wound up using gsutil directly because I couldn't find anything in those APIs for the concept of "please download this *entire* folder"--there's quite a lot in the Fuchsia SDK, and it'd be odd to have to enumerate each file.  Is there a "download this whole subdirectory" command I didn't see?  Or, is there a preference for e.g. zipping those files up and unzipping them once they're downloaded to Clusterfuzz?